plugins/com.google.cloud.tools.eclipse.appengine.whitelist.test/src/com/google/cloud/tools/eclipse/appengine/whitelist/XmlTest.java

```diff
@@ -50,4 +51,15 @@ public void testLoadExtensionClass() {
     }
   }
   
+  @Test
+  public void testLoadExtensionElements() {
+    NodeList extensions = document.getDocumentElement().getElementsByTagName("extension");
+    for (int i = 0; i < extensions.getLength(); i++) {
+      Element extension = (Element) extensions.item(i);
+      String point = extension.getAttribute("point");
+      Assert.assertTrue(point.length() > 0);
```

Comment:
Do you have a preference against static imports on Assert?
Ah, we can test the existence of the extension point with:
```
IExtensionRegistry registry = RegistryFactory.getRegistry();
IExtensionPoint extensionPoint = registry.getExtensionPoint(point);
Assert.assertNotNull(extensionPoint);
```